meta-security-smack/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -15,6 +15,21 @@ _ System -wx---
 ^ System::Run rwxat-
 ^ System rwxa--
 EOF
+
+    mkdir -p ${D}/${libdir}/tmpfiles.d
```

Comment:
Sure, can do that... but shouldn't ${sysconfdir}/smack/accesses.d at the beginning of the script be created with install -d, as well for consistency ?